csaf_2.0/json_schema/csaf_json_schema.json

```diff
@@ -18,11 +18,11 @@
         "properties": {
           "names": {
             "title": "List of acknowledged names",
-            "description": "Contains the names of entities being recognized.",
+            "description": "Contains the names of persons being recognized.",
```

Comment:
See my nit comments in the prose, in short: Maybe use the term `contributor` as core concept?